core/controlplane/config/config.go

```diff
@@ -113,6 +113,10 @@ func NewDefaultCluster() *Cluster {
 			MapPublicIPs:       true,
 			Experimental:       experimental,
 			ManageCertificates: true,
+			AmazonSsmAgent: AmazonSsmAgent{
+				Enabled:     false,
+				DownloadUrl: "https://github.com/DailyHotel/amazon-ssm-agent/releases/download/v2.0.805.1/ssm.linux-amd64.tar.gz",
```

Comment:
@ankon @c-knowles  First, AWS does not distribute AMD64 binaries for CoreOS so you have to build it by yourself. Second, URL might look suspicious but the company is very well-known in Korea, Republic of. This is one of the biggest company among the mobile hotel booking services. Anyway, as @ankon suggested, setting the default to an empty string might be better.